ArcGISRuntimeSDKQt_CppSamples/Geometry/ClipGeometry/ClipGeometry.cpp

```diff
@@ -0,0 +1,129 @@
+// [WriteFile Name=ClipGeometry, Category=Geometry]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "ClipGeometry.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+#include "Geometry.h"
+#include "Graphic.h"
+#include "GeometryEngine.h"
+#include "GraphicsOverlay.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleFillSymbol.h"
+
+using namespace Esri::ArcGISRuntime;
+
+ClipGeometry::ClipGeometry(QQuickItem* parent /* = nullptr */):
+  QQuickItem(parent)
+{
+}
+
+void ClipGeometry::init()
+{
+  // Register the map view for QML
+  qmlRegisterType<MapQuickView>("Esri.Samples", 1, 0, "MapView");
+  qmlRegisterType<ClipGeometry>("Esri.Samples", 1, 0, "ClipGeometrySample");
+}
+
+void ClipGeometry::componentComplete()
+{
+  QQuickItem::componentComplete();
+
+  // find QML MapView component
+  m_mapView = findChild<MapQuickView*>("mapView");
+
+  // Create a map using the topographic basemap
+  m_map = new Map(Basemap::topographic(this), this);
+
+  // Create a graphics overlay for the Colorado graphic
```

Comment:
Please update this comment now that these are grouped. Something like this should work.

```
// Create the graphics overlays
```